lib/container.js

```diff
@@ -283,8 +284,29 @@ function loadGherkinSteps(paths) {
   global.After = fn => event.dispatcher.on(event.test.finished, fn);
   global.Fail = fn => event.dispatcher.on(event.test.failed, fn);
 
-  for (const path of paths) {
-    loadSupportObject(path, `Step Definition from ${path}`);
+  // If gherkin.steps is string, then this will iterate through that folder and send all step def js files to loadSupportObject
+  // If gherkin.steps is Array, it will go the old way
+  // This is done so that we need not enter all Step Definition files under config.gherkin.steps
+  if (Array.isArray(paths)) {
+    for (const path of paths) {
+      loadSupportObject(path, `Step Definition from ${path}`);
+    }
+  } else {
+    let folderPath;
+    if (paths.charAt(0) === '.') {
+      folderPath = path.join(global.codecept_dir, paths);
+    }
+    if (folderPath !== '') {
+      glob(folderPath, (err, files) => {
```

Comment:
Can you use like `glob.sync(pattern, options).forEach((file) => {` ?

Reply:
I'm curious as to why does it have to be synchronous?

Reply:
Have changed the code as per recommendation.